internal/pkg/describe/lb_web_service.go

```diff
@@ -411,3 +471,20 @@ func appendServiceDiscovery(sds []*ServiceDiscovery, sd serviceDiscovery, env st
 	}
 	return sds
 }
+
+func appendServiceConnect(scs serviceConnects, dnsName string, env string) serviceConnects {
+	exist := false
+	for _, s := range scs {
+		if s.DNSName == dnsName {
```

Comment:
The short variable names make it a bit difficult to figure out what is being represented here due to all the acronyms.
Can we expand on our variable names?